### PR TITLE
SQL table

### PR DESCRIPTION
**Pull request by Daron976**
## In this PR:
- Added a species column.
- Inserted new data.
- Updated species column with transactions.
- Altered data with transactions.
- Wrote queries to answer various questions.
### Images
Screenshot from 2022-12-06 20-35-41
Screenshot from 2022-12-06 20-37-12
Screenshot from 2022-12-06 20-38-15
Screenshot from 2022-12-06 20-38-31
Screenshot from 2022-12-06 20-39-01
Screenshot from 2022-12-06 20-39-46
Screenshot from 2022-12-06 20-40-02
Screenshot from 2022-12-06 20-52-53
Screenshot from 2022-12-06 20-54-53
Screenshot from 2022-12-06 21-44-38
Screenshot from 2022-12-06 21-46-51
Screenshot from 2022-12-06 21-47-57
Screenshot from 2022-12-06 21-49-29
Screenshot from 2022-12-06 22-00-34
Screenshot from 2022-12-06 22-14-40
Screenshot from 2022-12-06 22-26-22

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
   date_of_birth date,
   escape_attempts int,
   neutered boolean,
-  weight_kg float
+  weight_kg float,
+  species varchar
 );
```
Comment (HAMMAS-SALEEM) [inline schema.sql]: - [ ] You are supposed to add species column with a different query but here you have added the species column in the previous query which is not what you are supposed to do. 😨 You can see below to get an idea what i am talking about 👍

```
CREATE TABLE animals (
  id int GENERATED BY DEFAULT AS IDENTITY,
  name varchar,
  date_of_birth date,
  escape_attempts int,
  neutered boolean,
  weight_kg float
);

ALTER TABLE animals ADD species varchar(255);
```

File: queries.sql [hunk 2]
```diff
@@ -28,4 +28,49 @@ WHERE name <> 'Gabumon';
...
+
+ROLLBACK TO sp1;
+
+BEGIN;
+
+UPDATE animals
+  SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT;
```
Comment (HAMMAS-SALEEM) [inline queries.sql]: These are transaction queries means these queries are manipulating the data in your database so these queries are supposed to be in data.sql file which is responsible for having data manipulation queries. queries.sql file is just responsible for data retrieval queries (means queries with Select) 👍

File: queries.sql [hunk 3]
```diff
@@ -28,4 +28,49 @@ WHERE name <> 'Gabumon';
```
Comment (HAMMAS-SALEEM) [inline queries.sql:28]: Capture

- [ ] You didn't make these queries. Kindly follow these requirements 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @Daron976 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Daron976) [reply queries.sql]: Please note that in the instructions I clearly say that these transactions should be in the queries.sql file
Screenshot from 2022-12-06 22-27-33

Comment (HAMMAS-SALEEM) [reply queries.sql]: Hey @Daron976 My bad. Its good that you have mentioned this 👍 
Thanks for your feedback. You can leave this change and work on the other changes 😊

**Review by t-yanick — APPROVED** [review]
Hi @Daron976, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**